api_app/analyzers_manager/file_analyzers/filescan.py

```diff
@@ -0,0 +1,73 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+import time
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from api_app.exceptions import AnalyzerRunException
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class FileScan(FileAnalyzer):
+    """Filescan Class"""
```

Comment:
```
 `FileScan_Upload_File` analyzer.
```

Reply:
Should i change the class docsrting from `"""Filescan Class"""` to ``FileScan_Upload_File` analyzer` or should i change class name from `FileScan` to `FileScan_Upload_File` ?

Reply:
my bad I wasn't clear.

- change the doc string to "`FileScan_Upload_File` analyzer"
- change class name to `FileScanUpload`. (also remember to change the `python_module` in JSON config)